### DISCUSSION TITLE
请问如何在创意工坊中加入自己的工具呢

### DISCUSSION THREAD

**Discussion by 6drf21e** [KeJunMao/ai-anything#10] [General]
先感谢下 @KeJunMao 的高产，  几天没来看发现更新了好多实用的功能 👍👍👍
----
现在遇到一个问题
后端mongodb已经调试好了，我手工insert数据可以在创意工坊看到。
但是如何将自己diy的工具加入到创意工坊呢？  自己添加的工具自己可以看到， 但是没有将自己的工具发布到创意工坊的办法。

**Comment by KeJunMao** [comment]
点击右上角即可通过github或者google登录，登录后在本地工具详情下面可以看 `upload to workshop` 按钮

如果已经上传了，那就是 `Sync to workshop`

ps: 有必要做一个文档网站吗..
